### DISCUSSION TITLE
What shade of white is 🦄 this unicorn?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#665] [Galaxy Trivia] [answered]
.

**Comment by DevElkami** [comment]
Uniwhite ? :laughing:

**Reply by SANKARAMDAS** [reply]
@DevElkami don't worry, I will provide some ....

Choose one from the list -
- Marshmallow
- Moonglow
- Eggshell

**Comment by ParanoidUser** [comment]
Great questions, **Sankaram**! 👏👏 Thank you for your active participation in the project. Keep them coming!

---
I guess for precise answer we need a hint on what platform / device [the unicorn emoji](https://emojipedia.org/unicorn) will be displayed on. Here is a few candidates :
 - Android: #E0E0E0 (Gray88/Gainsboro)
 - Samsung: #FFFFFF (White)
 - Microsoft: #EFD5FF (Blue Chalk)
 - Apple: #F9FAFD (Ghost White)

**Reply by SANKARAMDAS** [reply]
@ParanoidUser It's very funny. You indeed do some research.  
Okay, Choose one from the list- 

- Marshmallow
- Moonglow
- Eggshell

Hope You will choose the right one.

**Reply by ParanoidUser** [reply]
![image](https://user-images.githubusercontent.com/5120290/235709062-c0db2aab-0d8f-459f-b1c5-443584dd3a58.png)

**Reply by ParanoidUser** [reply]
Thank you for the hint! 😉

**Reply by SANKARAMDAS** [reply]
Any time.

**Comment by peterbaccaro** [comment, marked as answer]
The unicorn is **Marshmallow** white.
